client/tokenized-payment-request/frontend-utils.js

```diff
@@ -107,3 +126,126 @@ export const displayLoginConfirmationDialog = ( paymentRequestType ) => {
 		)?.redirect_url;
 	}
 };
+
+export const getErrorMessageFromNotice = ( notice ) => {
```

Comment:
```suggestion
/**
 * Parses HTML error notice and returns single error message.
 *
 * @param {string} notice Error notice DOM HTML.
 * @return {string} Error message content
 */
export const getErrorMessageFromNotice = ( notice ) => {
```
Doc comment maybe? I had to dig through the code to try to figure out what this function did: have I understood it correctly? 🤔

Reply:
This function was reused from the old implementation. Your comment seems correct to me. Thanks!